### PR TITLE
Update witch satellite with new character story

### PR DESCRIPTION
**Pull request by sganondorf**
Filled this out as an initial new direction for the witch character. 
Played through fine, let me know what you all think!

~Also: I did not change the effects on metrics because I think this still works.
Choose to help results in -2 trust, +1 charity while rejecting Amice results in +3 superstition.
Please let me know if you think those might need any tweaking as well!~

**New change!**
Helping the witch is now +2 superstition and +1 charity
Not helping is -2 superstition

### CODE REVIEW

File: data/accidentalProphetStory.xml [hunk 1]
```diff
@@ -333,24 +333,24 @@ They share an uproarious laugh over what they figure got him into that state.</e
...
 				<prominence id="trustTheme">1</prominence>
 				<prominence id="charityTheme">1</prominence>
 			</functionalDescription>
 			<choice>
-				<text>Help her by ensuring her path is clear.</text>
+				<text>Agree to accompany her.</text>
 				<outcome>
-					<text>Seeing you help the witch, your followers grow distrustful of you despite your giving nature.</text>
+					<text>While a disgruntled crowd follows as you set out, Amice is not harassed further. As she collects the moss, she muses about it's purposes for helping with headaches, particularly those brought on by visions. When you ask her about the visions, she smiles knowingly but only response that perhaps it's a tale best left for another time.</text>
 					<quantModifier id="trustMetric" absolute="false">-2</quantModifier>
```
Comment (gailcarmichael) [inline data/accidentalProphetStory.xml]: Typo: `it's purpose` -> `its purpose`

Comment (gailcarmichael) [reply data/accidentalProphetStory.xml]: `only response` -> `only responds`?

**Comment by gailcarmichael** [comment]
I wonder if helping the witch could actually increase superstition? (Maybe it depends on what we mean by superstition - of whom and by whom; something to discuss when we next meet!)

File: data/accidentalProphetStory.xml (hunk 1 above)
Comment (sganondorf) [reply data/accidentalProphetStory.xml]: Thanks for catching these, will fix! 👍

**Pull request merged**